### ISSUE TITLE
My computer freezes when transcripts process starts

### ISSUE THREAD

**Issue by llurk** [Purfview/whisper-standalone-win#254]
For some reason, I'm currently unable to use Faster Whisper on SE and the local app. Whenever I initiate the transcript process, my computer freezes. This only happens when Faster Whisper utilizes the GPU; I'm unsure if it occurs with the CPU. Additionally, I can fully utilize Const-Me with the large model on SE.
This occurs with both Faster Whisper and the XXL version.

**Comment by Purfview** [comment]
Are you sure that it runs on GPU?
What changed on your PC before it started to freeze?

**Comment by llurk** [comment]
In the terminal, it's noted that the software runs on CUDA. Usually, the computer freezes immediately when the GPU starts processing the initial characters, although sometimes it might run a few lines. Initially, I thought it was a hardware issue and reinstalled Windows and drivers several times, but recently, when I switched to Const-Me with the large model or ran benchmarks, my computer worked fine. What I noticed when Whisper runs through MSI Afterburner is that initially, the GPU clock speed drops very low, then when the GPU starts computing, it goes to full load and immediately freezes the system.
